javascripts/src/dataclips.js

```diff
@@ -199,10 +206,15 @@ export default class Dataclips {
                 metadata: { style: xlsx_number_formats.datetime_formatter.id },
               };
             case "duration":
-              return {
-                value: value.as("day"),
-                metadata: { style: xlsx_number_formats.duration_formatter.id },
-              };
+              if (disableSeconds) {
+                return {
+                  value: value.values["hours"] + ":" + formatZeros(Math.abs(value.values["minutes"])),
+                };
+              } else {
```

Comment:
if disableSeconds is false than it should be behavior as it was so duration_formatter instead

Reply:
I can rollback the changes but my solution solves the wrong (in my opinion) master behavior.
Below screens to compare. As you can see, on master we display sometimes milliseconds...


<img width="172" alt="Zrzut ekranu 2022-11-15 o 08 36 57" src="https://user-images.githubusercontent.com/49039289/201858376-b36b0c7e-75e8-4a05-a32b-c5b95c209b22.png">
<img width="137" alt="Zrzut ekranu 2022-11-15 o 08 37 42" src="https://user-images.githubusercontent.com/49039289/201858382-0ec0aa96-8c9d-4e32-b7b5-3990149b455d.png">

Reply:
please do not extend scope of this task, so please revert.
Issues are not part of this task , please do it separately in different one with proper description so everyone knows what is the reason of change etc...